apps/condo/domains/ticket/schema/TicketReportService.js

```diff
@@ -38,29 +65,34 @@ const TicketReportService = new GQLCustomSchema('TicketReportService', {
             access: true,
             type: 'type TicketReportWidgetOutput { data: [ TicketReportData! ] }',
         },
+        {
+            access: true,
+            type: `enum TicketType { ${TICKET_TYPES.join(' ')} }`,
+        },
+        {
+            access: true,
+            type: `enum ChartViewMode { ${CHART_VIEW_MODES.join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input TicketReportAnalyticsInput { dateFrom: String!, dateTo: String!, groupBy: String!, userOrganizationId: String!, ticketType: TicketType!, viewMode: ChartViewMode!, addressList: [ String! ] }',
```

Comment:
userOrganizationId? use already defined types. Look at `createTicket` GQL

Reply:
`addressList` ? why not IDs?

Reply:
> userOrganizationId? use already defined types. Look at `createTicket` GQL

@pahaz This is just organizationId

Reply:
> `addressList` ? why not IDs?

there it is planned to pass an array of ids